tests/integration/test_fim/test_registry/test_registry_wildcards/test_registry_wildcards.py

```diff
@@ -0,0 +1,301 @@
+
+
+# Tests
+@pytest.mark.parametrize('configuration, metadata', zip(t1_configurations, t1_configuration_metadata), ids=t1_case_ids)
+def test_registry_key_wildcards(configuration, metadata, set_wazuh_configuration, truncate_monitored_files,
+                                configure_local_internal_options_function, restart_wazuh_function,
+                                wait_syscheck_start):
+    '''
+    description: Check the behavior of FIM when using wildcards to configure the path of registry keys, and validate
+                 the keys creation, modification and deletion is detected correctly.
+
+    wazuh_min_version: 4.5.0
+
+    test_phases:
+        - setup:
+            - Set wazuh configuration.
+            - Clean logs files and restart wazuh to apply the configuration.
+        - test:
+            - Check that one or more keys are detected when the configured wildcard is expanded
+            - Create a subkey inside the first monitored key and check
+            - Wait for scan and check subkey has been detected as 'added'
+            - Modify the subkey
+            - Wait for scan and check subkey has been detected as 'modified'
+            - Delete the subkey
+            - Wait for scan and check subkey has been detected as 'deleted'
+        - teardown:
+            - Restore configuration
+            - Stop wazuh
+
+    tier: 1
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Configuration values for ossec.conf.
+        - metadata:
+            type: dict
+            brief: Test case data.
+        - set_wazuh_configuration:
+            type: fixture
+            brief: Set ossec.conf configuration.
+        - configure_local_internal_options_function:
+            type: fixture
+            brief: Set local_internal_options configuration.
+        - policies_file:
+            type: string
+            brief: path for audit policies file to use on restore_win_whodata_policies fixture
+        - restore_win_whodata_policies
+            type: fixture
+            brief: restores windows audit policies using a given csv file after yield
+        - restart_syscheck_function:
+            type: fixture
+            brief: restart syscheckd daemon, and truncate the ossec.log.
+        - wait_for_fim_start_function:
+            type: fixture
+            brief: check that the starting fim scan is detected.
+
+    assertions:
+        - One or more keys have been configured after wildcard expansion
+        - Assert 'registry_key added' event has been detected
+        - Assert 'registry_key modified' event has been detected
+        - Assert 'registry_key deleted' event has been detected
+
+    input_description:
+        - The file 'configuration_registry_wildcards.yaml' contains the configuration template for the test.
+        - The file 'cases_registry_key_wildcards.yaml' contains test case descriptions, configuration values and
+          metadata for each case.
+
+
```

Comment:
```suggestion
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4220/commits/cc756e243308370882f0aab65b88291a92535154